twenty-first/src/math/polynomial.rs

```diff
@@ -347,6 +353,28 @@ where
         Self::new(hadamard_product)
     }
 
+    pub fn batch_multiply(factors: &[Self]) -> Self {
+        factors.iter().fold(Self::one(), |l, r| l.multiply(r))
+    }
```

Comment:
Requires a doc comment.

Reply:
Fixed in c0fb4655.

It raises the question, at which point is a function's name so obvious and elucidating that a docstring cannot possibly add new information?

Reply:
Fair question. Maybe this method does not need a doc comment after all; looking at the signature now, I'm not sure why I thought it needed one.